src/features/workspace/chat/context/ChatContext.tsx

```
@@ -87,25 +87,56 @@ function chatReducer(state: ChatState, action: ChatAction): ChatState {
       };
     case 'ADD_MESSAGE':
       const existingMessages = state.messages[action.payload.chatRoomId] || [];
-      const messageExists = existingMessages.some(msg => msg.id === action.payload.message.id);
+      const newMessage = action.payload.message;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)